framework/src/controller/bus.ts

```diff
@@ -251,6 +268,10 @@ export class Bus {
 	public async cleanup(): Promise<void> {
 		this._emitter.removeAllListeners();
 		this._ipcServer.stop();
+
+		if (this.config.rpc.enable && this.config.rpc.mode === 'ws') {
```

Comment:
instead of checking `enable` and `ws` maybe it's better to check existence of `this._WSServer`

Reply:
Done